serverless.yml

```diff
@@ -13,6 +13,11 @@ provider:
       Action:
         - "ses:SendEmail"
       Resource: "*"
+    - Effect: "Allow"
+      Action:
+        - "s3:PutObject"
+      Resource: 
+        - arn:aws:s3:::${file(env.${opt:stage}.yml):S3_LOGGING_BUCKET}/*
```

Comment:
This is gonna be tricky to fix, but this line effectively makes it impossible to disable S3 logging as serverless will now require this variable to be set. It might be possible to fix this via javascript references (https://serverless.com/framework/docs/providers/aws/guide/variables#reference-variables-in-javascript-files), but I haven't checked whether this works.

Reply:
I had that thought as well so I was sure to test it. You can effectively disable S3 logging like so:

```LOG_JSON_TO_S3: false
S3_LOGGING_BUCKET: ""```

An `S3_LOGGING_BUCKET` variable must be present, but it doesn't need to have a valid bucket name in it. I think this is OK, if not 100% ideal.